YOLayout/UIView+YOView.h

```diff
@@ -0,0 +1,22 @@
+//
+//  UIView+YOView.h
+//  YOLayoutExample
+//
+//  Created by Lucas Yan on 8/1/16.
+//  Copyright © 2016 YOLayout. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@class YOLayout;
+
+@interface UIView (YOView)
+
+@property (nonatomic) YOLayout *layout;
+
++ (void)initLayout;
```

Comment:
Maybe call this something more indicative of what's happening. Perhaps `injectYOLayout` or `swizzleLayoutMethods` or `useYOLayout`